README.md

```diff
@@ -144,7 +144,19 @@ The next time you invoke a documentation build with the "Build Documentation"
 button in Xcode's Product menu, your custom `docc` will be used for the build.
 You can confirm that your custom `docc` is being used by opening the latest build
 log in Xcode's report navigator and expanding the "Compile documentation" step.
-  
+
+### Invoking `docc` from Swift Package Manager
+
+You can also test a locally built version of Swift-DocC using Swift Package Manager
+CLI. SwiftPM will try to read `DOCC_EXEC` environment variable value, and use
```

Comment:
What do you think of avoiding the CLI abbreviation here? I think it's possible some reader's won't be familiar with it.

```suggestion
You can also test a locally built version of Swift-DocC using the Swift Package Manager from the command line.
SwiftPM will try to read `DOCC_EXEC` environment variable value, and use
```